lib/evss/documents_service.rb

```diff
@@ -13,5 +13,18 @@ def initialize(vaafi_headers = {})
     def all_documents
       get 'documents/getAllDocuments'
     end
+
+    def upload(file_name, file_body, claim_id, tracked_item_id)
+      post 'queuedDocumentUploadService/ajaxUploadFile', file_body do |req|
+        req.headers['Content-Type'] = 'application/octet-stream'
+        req.params['systemName'] = 'vets.gov'
+        # TODO: Get real doctypes/descriptions
+        req.params['docType'] = 'L023'
+        req.params['docTypeDescription'] = 'Other Correspondence'
+        req.params['claimId'] = claim_id
+        req.params['trackedItemIds'] = tracked_item_id
```

Comment:
I think there was some discussion on this in Slack but I forgot what the outcome was. This can be an array or a single item? Can you add a comment to this, the mapped param name makes it seem like we should be sending an array.